pkg/query-service/auth/jwt.go

```diff
@@ -47,11 +50,18 @@ func validateUser(tok string) (*model.UserPayload, error) {
 	if !claims.VerifyExpiresAt(now, true) {
 		return nil, model.ErrorTokenExpired
 	}
```

Comment:
orgId claim will not be present in the older jwt tokens hence the extra handling